_layouts/page.html

```diff
@@ -4,5 +4,6 @@
 
 <div class="page">
   <h1 class="page-title">{{ page.title }}</h1>
-  {{ content }}
+  <a href="{{ site.baseurl }}{{ page.url }}"></a>
```

Comment:
@mafagafogigante -- how would you format the link?

I think it's correct :-) See *Rule 1* here: [Configuring Jekyll for User and Project GitHub Pages](http://downtothewire.io/2015/08/15/configuring-jekyll-for-user-and-project-github-pages/)